### PR TITLE
implement tokens for sw-card component

### PR DESCRIPTION
**Pull request by Haberkamp**
## What?

I added design tokens to the `sw-card` component.

## Why?

This makes the component support multiple themes.

## How?

I replaced the SCSS variables with CSS custom properties that use our Meteor Tokens.

## Testing?

You can use the preview branch to check the results

### CODE REVIEW

File: packages/component-library/src/components/layout/sw-card/sw-card.vue [hunk 1]
```diff
@@ -248,20 +236,15 @@ export default defineComponent({
...
-    color: $color-darkgray-300;
+    color: var(--color-text-primary-default);
     font-size: $font-size-large;
     font-weight: $font-weight-semi-bold;
     line-height: 18px;
   }
 
   .sw-card__subtitle {
-    color: $color-darkgray-50;
+    color: var(--color-text-secondary-default);
```
Comment (fabianhueske) [inline packages/component-library/src/components/layout/sw-card/sw-card.vue:247]:
```suggestion
    color: var(--color-text-tertiary-default);
```

File: packages/component-library/src/components/layout/sw-card/sw-card.vue [hunk 2]
```diff
@@ -305,21 +286,13 @@ export default defineComponent({
...
-
   .sw-card__content {
     display: flow-root;
     flex-basis: 100%;
     padding: 30px;
-    background: $color-white;
     background-clip: padding-box;
-    border-radius: $border-radius-lg;
     position: relative;
+    color: var(--color-text-secondary-default);
```
Comment (fabianhueske) [inline packages/component-library/src/components/layout/sw-card/sw-card.vue:295]:
```suggestion
    color: var(--color-text-primary-default);
```

**Review by fabianhueske — APPROVED** [review]

File: packages/component-library/src/components/layout/sw-card/sw-card.stories.ts [hunk 3]
```diff
@@ -103,7 +103,7 @@ export const ExtendedStory: StoryObj<SwCardMeta> = {
     },
     template: `
     <sw-card v-bind="args">
-        <h4>
+        <h4 style="color: var(--color-text-primary-default);">
```
Comment (jleifeld) [inline packages/component-library/src/components/layout/sw-card/sw-card.stories.ts:106]: I would prefer to have it inside the scss part

**Review by jleifeld — APPROVED** [review]
Looks good. Only one small thing

**Pull request merged**